custom/icds_reports/reports/adolescent_girls.py

```diff
@@ -23,8 +24,10 @@ def get_data_for(filters):
         ).values(
             '%s_name' % loc_level, '%s_map_location_name' % loc_level
         ).annotate(
-            valid=Sum('cases_person_adolescent_girls_11_14'),
-            all=Sum('cases_person_adolescent_girls_11_14_all'),
+            valid=Sum('cases_person_adolescent_girls_11_14_out_of_school') if beta else Sum(
```

Comment:
there is a lot of `something if beta else something_else` over the code.
Would it be better to get that information up front in the methods they are being done in, for example in this one,
```
def get_adolescent_girls_data_map(domain, config, loc_level, show_test=False, beta=False):
    if beta:
        annotate_field_for_valid = 'cases_person_adolescent_girls_11_14_out_of_school'
        # ...
     else:
        annotate_field_for_valid = 'cases_person_adolescent_girls_11_14'
        # ...
 ```
and then later just call
```
            valid=Sum(annotate_field_for_valid)
```
I am not sure if this makes a good enough difference but just wanted to suggest this if it makes sense to you.